### PR TITLE
Enhance helm-chart to support Promscale trace support

### PR DESCRIPTION
**Pull request by VineethReddy02**

### CODE REVIEW

File: helm-chart/README.md [hunk 1]
```diff
@@ -109,7 +109,8 @@ helm install --name my-release -f myvalues.yaml .
 | `connection.port`                 | Port the db listens to                      | `5432`                             |
 | `connection.dbName`               | Database name in TimescaleDB to connect to  | `timescale`                        |
 | `connection.sslMode`              | SSL mode for connection                     | `require`                          |
-| `service.port`                    | Port the connector pods will accept connections on | `9201`                      |
```
Comment (paulfantom) [inline helm-chart/README.md:112]: We should probably mention that we are breaking backwards compatibility with this

Comment (VineethReddy02) [reply helm-chart/README.md:112]: Added it to the draft release notes. As we discussed.

**Review by paulfantom — APPROVED** [review]
lgtm

**Pull request merged**